kornia/enhance/adjust.py

```diff
@@ -420,56 +429,84 @@ def _blend_one(input1: torch.Tensor, input2: torch.Tensor, factor: Union[float,
     return torch.stack([_blend_one(input[i], result[i], factor[i]) for i in range(len(factor))])
 
 
+# Code taken from: https://github.com/pytorch/vision/pull/796
+def _scale_channel(im):
+    """Scale the data in the channel to implement equalize."""
+    im = im * 255
```

Comment:
If ```_scale_channel``` is extracted as a private function than nested inside as previously, should we also add some checks there?

e.g. ```im``` must be (CHW) or (CDHW) and ````min(im), max(im)``` must be in (0, 1)